src/Cli/Microsoft.TemplateEngine.Cli/Commands/list/BaseListCommand.cs

```diff
@@ -72,4 +80,12 @@ protected override Task<NewCommandStatus> ExecuteAsync(
         protected override ListCommandArgs ParseContext(ParseResult parseResult) => new(this, parseResult);
 
     }
+
+#pragma warning disable SA1201 // Elements should appear in the correct order
+    public enum FormatOptions
+    {
+        Console,
```

Comment:
I think the current layout would be more accurately described as `'table'` instead of '`console'`, what do you think?